src/test/java/com/expediagroup/hiveberg/TestIcebergSerDe.java

```diff
@@ -84,7 +89,95 @@ public void testDeserializer() throws IOException, SerDeException {
     reader.next(null, value);
 
     IcebergSerDe serde = new IcebergSerDe();
-    Object row = serde.deserialize(value);
-    //TODO: Finish this test with assertions
+    List<?> deserialized = (List<?>) serde.deserialize(value);
```

Comment:
I'm not sure the `?` adds anything here. Could you just use `Object` instead? Or does that mean you need to do additional casting below?